.github/CODEOWNERS

```diff
@@ -83,7 +83,7 @@
 /sdk/communication/communication-network-traversal/ @ajpeacock0 @nathpete-msft
 
 # PRLabel: %Communication - Rooms
-/sdk/communication/communication-rooms/ @anujissarMS
+/sdk/communication/communication-rooms/ @anujissarMS @Azure/acs-rooms-sdk
```

Comment:
Do we need to keep anujissarMS as owner?

Reply:
Doesn't hurt to have an extra pair of eyes :)